packages/menu/src/useMenu.ts

```diff
@@ -729,12 +749,39 @@ export const useMenu = <T extends HTMLElement = HTMLElement, M extends HTMLEleme
         'aria-checked': selected,
         'aria-disabled': itemDisabled,
         role: itemRole === null ? undefined : itemRole,
+        href,
         onClick,
         onKeyDown,
         onMouseEnter,
         ...other
       };
 
+      /**
+       * Validation
+       */
+
+      if (href && (isNext || isPrevious || type)) {
+        let invariantKey: string;
+
+        if (isNext) invariantKey = 'isNext';
+        else if (isPrevious) invariantKey = 'isPrevious';
+        else invariantKey = type!;
+
+        const invariantType = {
+          isNext: 'isNext',
+          isPrevious: 'isPrevious',
+          radio: 'radio',
+          checkbox: 'checkbox'
+        }[invariantKey];
+
+        throw new Error(`Error: Anchor item '${value}' can't use '${invariantType}'`);
```

Comment:
Can we get "useMenu" incorporated in this message? Compare with https://github.com/zendeskgarden/react-containers/blob/8f4195f0464efd43e905c367f1c52d8a93d69608/packages/combobox/src/useCombobox.ts#L157-L171